living_documentation_regime/model/consolidated_issue.py

```diff
@@ -200,18 +195,13 @@ def generate_directory_path(self, issue_table: str) -> list[str]:
             # If no label ends with "Topic", create a "NoTopic" issue directory path
             if not topic_labels:
                 self.__topics = ["NoTopic"]
-                logger.error("No Topic label found for Issue #%i: %s (%s)", self.number, self.title, self.repository_id)
+                self.__errors.update({"TopicError": "No Topic label found."})
+
                 no_topic_path = os.path.join(output_path, "NoTopic")
                 return [no_topic_path]
 
             if len(topic_labels) > 1:
-                logger.debug(
-                    "Multiple Topic labels found for Issue #%s: %s (%s): %s",
-                    self.number,
-                    self.title,
-                    self.repository_id,
-                    ", ".join(topic_labels),
-                )
+                self.__errors.update({"TopicError": "More than one Topic label found."})
```

Comment:
Add where we found more topics.

Reply:
This is directly connected to the correct consolidated_issue. In the report page is written exactly the issue with the error message and error type.